Packages/io.chainsafe.web3-unity/package.json

```diff
@@ -1,32 +1,33 @@
 {
-    "name": "io.chainsafe.web3-unity",
-    "version": "2.5.5",
-    "displayName": "web3.unity SDK",
-    "description": "web3.unity is an open-source gaming SDK written in C# and developed by ChainSafe Gaming. It connects games built in the Unity game engine to the blockchain. The library currently supports games built for web browsers (WebGL), iOS/Android mobile, and desktop. web3.unity is compatible with most EVM-based chains such as Ethereum, Polygon, Moonbeam, Cronos, Nervos, and Binance Smart Chain, letting developers easily choose and switch between them to create the best in-game experience.",
-    "license": "LGPL-3.0-only",
-    "licensesUrl": "https://github.com/ChainSafe/web3.unity/blob/main/LICENSE",
-    "documentationUrl": "https://docs.gaming.chainsafe.io/",
-    "dependencies": {
-        "com.unity.nuget.newtonsoft-json": "3.0.2",
-        "com.unity.textmeshpro": "3.0.6"
-    },
-    "keywords": [
-        "web3",
-        "ethereum",
-        "evm",
-        "blockchain",
-        "nft"
-    ],
-    "author": {
-        "name": "ChainSafe Gaming",
-        "email": "bd@chainsafe.io",
-        "url": "https://gaming.chainsafe.io/"
-    },
-    "samples": [
-        {
-            "displayName": "Web3.Unity Samples",
-            "description": "Contains implementations for Web3 Sample Operations using Web3.Unity.",
-            "path": "Samples~/Web3.Unity"
-        }
-    ]
-}
+  "name": "io.chainsafe.web3-unity",
+  "version": "2.5.5",
+  "displayName": "web3.unity SDK",
+  "description": "web3.unity is an open-source gaming SDK written in C# and developed by ChainSafe Gaming. It connects games built in the Unity game engine to the blockchain. The library currently supports games built for web browsers (WebGL), iOS/Android mobile, and desktop. web3.unity is compatible with most EVM-based chains such as Ethereum, Polygon, Moonbeam, Cronos, Nervos, and Binance Smart Chain, letting developers easily choose and switch between them to create the best in-game experience.",
+  "license": "LGPL-3.0-only",
+  "licensesUrl": "https://github.com/ChainSafe/web3.unity/blob/main/LICENSE",
+  "documentationUrl": "https://docs.gaming.chainsafe.io/",
+  "dependencies": {
+    "com.unity.nuget.newtonsoft-json": "3.0.2",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.walletconnect.modal": "1.1.6"
```

Comment:
aha so this will get the dependency itself?

Reply:
yeah, this references the UPM package for their solution